.eslintrc.yml

```diff
@@ -29,7 +29,7 @@ rules:
       objects: always-multiline
       imports: always-multiline
       exports: always-multiline
-      functions: never
+      functions: always-multiline
```

Comment:
This matches the `all` property value of Prettier https://prettier.io/docs/en/options.html#trailing-commas

Reply:
Got it! Thanks: I prefer this but couldn't find it.